src/controller/authentication-controller.ts

```diff
@@ -150,7 +151,7 @@ export default class AuthenticationController extends BaseController {
     const body = req.body as AuthenticationMockRequest;
 
     // Only allow in development setups
-    if (process.env.NODE_ENV !== 'development') return false;
+    if (process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test') return false;
```

Comment:
Won't this always be false

Reply:
No, because when you are in a `production` environment, you cannot perform a mock login. In `development` (your local environment) and `test` (when running test suites) this endpoint should be enabled.